docs/src/main/sphinx/language/types.rst

```diff
@@ -5,10 +5,32 @@ Data types
 Trino has a set of built-in data types, described below.
 Additional types can be provided by plugins.
 
-.. note::
+.. _type-mapping-overview:
+
+Trino type support and mapping
+------------------------------
+
+Connectors to data sources are not required to support all Trino data types
+described on this page. If there are data types similar to Trino's that are used
+on the data source, the connector may map the Trino and remote data types to
+each other as needed.
+
+Depending on the connector and the data source, type mapping may apply
+in either direction as follows:
+
+* **Data source to Trino** mapping applies to read and write operations, when
+  columns with specific types in the data source exist and are read or written
+  from Trino. In this type mapping table, all supported types in the data
+  source should be listed. Multiple remote data types may map to the same Trino
```

Comment:
```suggestion
  from Trino. Multiple remote data types may map to the same Trino
```

That sentence feels like instruction to doc authors instead of users.

Reply:
Same in the below paragraph.